src/libraries/ERC7739Utils.sol

```diff
@@ -75,16 +71,20 @@ library ERC7739Utils {
                 bytes1 current = buffer[i - 1];
                 if (current == bytes1(")")) {
                     // we found the end of the contentsName
-                    contentsName = LibString.slice(contentsDescr, i, buffer.length);
-                    contentsType = LibString.slice(contentsDescr, 0, i);
-                    return (contentsName, contentsType);
+                    return (string(buffer[i:]), string(buffer[:i]));
                 } else if (_isForbiddenChar(current)) {
                     // we found an invalid character (forbidden) - passthrough (fail)
                     break;
                 }
             }
         }
-        return ("", "");
+        // If we didn't find a valid contentsName, return empty strings
+        assembly ("memory-safe") {
+            contentsName.offset := 0
+            contentsName.length := 0
+            contentsType.offset := 0
+            contentsType.length := 0
+        }
```

Comment:
Return empty strings (check this plz :))

Reply:
Yeah, makes sense to me, it should just be no data